package.json

```diff
@@ -16,7 +16,7 @@
 		"arweave-bundles": "^1.0.3",
 		"arweave-mnemonic-keys": "^0.0.9",
 		"base64-js": "^1.5.1",
-		"commander": "^8.1.0",
+		"commander": "^8.2.0",
```

Comment:
So what happened was that I was missing commander APIs when trying to use the `program` instance and then I realized that it was because you had an abstraction layer in front of the commander types. Updating commander was just a head check that it wasn't that. It wasn't the best idea, but it seems to have worked out fine. From their release notes:
```
8.2.0 (2021-09-10)
Added
.showSuggestionAfterError() to show suggestions after unknown command or unknown option (#1590)
add Option support for values from environment variables using .env() (#1587)
Changed
show error for unknown global option before subcommand (rather than just help) (#1590)
Removed
TypeScript declaration of unimplemented Option method argumentRejected
```
So nothing major.